projects/plugins/jetpack/extensions/blocks/sharing-button/style.scss

```diff
@@ -82,14 +88,6 @@ a.jetpack-sharing-button__button {
 	display: none;
 }
 
-.jetpack-sharing-buttons__service-label.style-icon {
-	display: none;
-}
```

Comment:
This didn't seem necessary; that specific selector doesn't exist. I believe the correct one would be `.jetpack-sharing-button__service-label` (without the s), and that style is applied just above.